content/3_Silver/Binary_Search.problems.json

```diff
@@ -91,6 +104,32 @@
         "hasHints": true
       }
     },
+    {
+      "uniqueId": "usaco-1447",
+      "name": "Deforestation",
+      "url": "https://usaco.org/index.php?page=viewproblem2&cpid=1447",
```

Comment:
this would be better in priority queues (if going off the official analysis)?

Reply:
sure